src/docs/user/references/Notification-service.adoc

```diff
@@ -78,6 +79,21 @@ The _Submitted by_ switch may be activated to be notified of events for job subm
 
 To modify a job subscription simply click on it, do your adjustments and save it.
 
+[[channels_subscription]]
+===== Channel subscriptions
+
+Like service subscriptions, channel subscriptions are unique to each user. It allows users to subscribe to channels created beforehand.
+A user that is a notification service administrator can subscribe to any channel. Otherwise either the channel is authorized
+for all user groups to subscribe or the user is part of at least one of the authorized groups of the channel.
+
+To visualize past notifications and start receiving new ones, a user must update his channel subscription
```

Comment:
a user must update his channel subscription with the following actions:
1) Activate a notification method for the channel subscription
2) Select desired channels from the user's available channels list
3) Finally, activate the subscription